feature-libs/user/public_api.ts

```diff
@@ -0,0 +1,5 @@
+/*
+ * Public API Surface of user
+ */
+export * from '@spartacus/user/account';
```

Comment:
I think we decided to not reexport symbols from grouping entry points and ideally have one way to import something.
What we can do here, is to add one module grouping all lib parts together, and export this.